app.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "rasa_nlu",
+  "name": "rasa",
```

Comment:
shall we just remove app.json while we are at it? we know the free dynos are too small to run it anyway